kingpin/actors/base.py

```diff
@@ -457,9 +457,7 @@ def _fetch(self, url, post=None, auth_username=None, auth_password=None):
 
         try:
             body = json.loads(http_response.body)
-        except ValueError as e:
-            raise exceptions.UnparseableResponseFromEndpoint(
-                'Unable to parse response from remote API as JSON: %s' % e)
+        except ValueError:
+            raise gen.Return(http_response.body)
```

Comment:
So that if the response is not JSON it will return the raw text. Similar to your support class.